src/compose/ComposeBox.js

```diff
@@ -444,6 +446,7 @@ class ComposeBox extends PureComponent<Props, State> {
     }
 
     const placeholder = getComposeInputPlaceholder(narrow, ownUserId, allUsersById);
+    const accessibilityLabel = narrowUiDescription(narrow, _);
```

Comment:
This shouldn't be called `accessibilityLabel` because it's not the entire accessibility label. Let's call it something like `narrowDescription`.